src/custom/constants/ipfs.ts

```diff
@@ -0,0 +1,6 @@
+export const PINATA_API_KEY = process.env.REACT_APP_PINATA_API_KEY as string
+export const PINATA_SECRET_API_KEY = process.env.REACT_APP_PINATA_SECRET_API_KEY as string
+
+export function getIpfsUri(): string {
+  return process.env.REACT_APP_IPFS_URI || 'https://ipfs.infura.io:5001/api/v0'
```

Comment:
why a function and not just an exported const? process.env won't change.

Also, feels weird to expose a function in `constants/ipfs` file